python/setup.py.in

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+from setuptools import setup
+
+
+setup(
+    name='ola',
+    version='@VERSION@',
+    author='Simon Newton',
+    author_email='nomis52@gmail.com',
+    description='OLA Python client bindings.',
+    license='GPL',
```

Comment:
The note for this says the license should only be used when "the license is not a selection from the “License” Trove classifiers. See the Classifier field. Notice that there’s a licence distribution option which is deprecated but still acts as an alias for license.":
https://docs.python.org/2/distutils/setupscript.html#additional-meta-data

Either way, the client should be LGPL 2.1 or greater (rather than GPL):
https://github.com/OpenLightingProject/ola/blob/master/python/ola/OlaClient.py#L2-L4

Reply:
E.g. https://github.com/codespell-project/codespell/blob/master/setup.py